src/main/java/logaggregatortool/LogAggregatorTool.java

```diff
@@ -16,10 +16,9 @@ public static void main(String[] args) {
         }
         String folderPath = args[0];
         File userInputFolderPath = new File(folderPath);
```

Comment:
What is the use of this line 18?
```suggestion
        File userInputFolderPath = new File(folderPath);
```

Reply:
done